client/src/Pages/HomeStyles.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components'
+
+const SearchButtonDiv = styled.div`
+  background: #282c34;
+  margin-top : 20px;
```

Comment:
Never use px as a unit for CSS styles. This is because px is hard-coded, and would be the same size regardless of what screen you're viewing this on. This can be problematic on different phone sizes.

Instead, you should be using `vh` or `vw`. For example, `10vh` would mean 10 percent of the window height, whereas `10vw` would mean 10 percent of the window width.